WordPress/Classes/Plugins/Views/PluginDetailsView.swift

```diff
@@ -98,7 +98,13 @@ struct PluginDetailsView: View {
             .listSectionSeparator(.hidden)
 
             if viewModel.isUninstalling {
-                uninstallingView()
+                inlineProgressView(title: Strings.uninstallingTitle, message: Strings.uninstallingMessage)
```

Comment:
Replacing the "Activate" button with a progress view in place should be enough. I wouldn't brother adding this separate view and titles. It looks a bit abrupt too.

Reply:
Good idea. I can look into this and create separate PRs for it if needed.